Doc/c-api/init.rst

```diff
@@ -373,6 +373,14 @@ Initializing and finalizing the interpreter
    :c:func:`Py_Initialize` is called again.
 
 
+.. c:function:: int Py_IsFinalizing()
+
+   Return non-zero if the Python interpreter is :term:`shutting down
+   <interpreter shutdown>`, return 0 otherwise.
```

Comment:
For consistency with `Py_IsInitialized()`:
```suggestion
   Return true (nonzero) if the Python interpreter is :term:`shutting down
   <interpreter shutdown>`, false (zero) if not.
```

Reply:
Ok, done